zyte_spider_templates/params.py

```diff
@@ -108,6 +108,43 @@ class MaxRequestsParam(BaseModel):
     )
 
 
+class SearchQueriesParam(BaseModel):
+    search_queries: List[str] = Field(
+        title="Search Queries",
+        description=(
+            "Turn the input URLs into search requests for these queries. You "
+            "may specify a separate search query per line. If search request "
+            "building fails, you can instead pass search URLs as input start "
+            "URLs, or customize the AI spider project with a search request "
+            "template page object (check the docs)."
```

Comment:
> Turn the input URLs into search requests for these queries.

This reads a bit cryptic to me :( 

I wonder if it's ok to explain it on a lower level, closer to the implementation: a list of search queries to submit using the search form found on the input URL.

Reply:
> or customize the AI spider project with a search request template page object (check the docs).

While technically correct, this applies to everything - e.g. to all the crawl strategies we provide, and to the extraction as well. I'd leave it out from the description; we need to communicate it, but we need to do it for everything somehow, so probably we should do it separately.

Reply:
5763840